sw/device/silicon_creator/mask_rom/mask_rom.c

```diff
@@ -156,6 +159,11 @@ static rom_error_t mask_rom_init(void) {
 
   sec_mmio_check_values(rnd_uint32());
   sec_mmio_check_counters(/*expected_check_count=*/1);
+
+  uint32_t ibex_secure_seed_got;
+  CSR_READ(CSR_REG_SECURESEED, &ibex_secure_seed_got);
+  HARDENED_CHECK_EQ(ibex_secure_seed_got, ibex_secure_seed);
+
```

Comment:
nit: should we move this to `mask_rom_pre_boot_check()`? Once we have all the checks we want there, we could implement something that uses shares, maybe?

Reply:
Ended up removing this check. I didn't realize that this CSR returns `0s` on reads.